jetty-util/src/main/java/org/eclipse/jetty/util/resource/Resource.java

```diff
@@ -418,11 +418,24 @@ public abstract boolean renameTo(Resource dest)
     /**
      * Returns the resource contained inside the current resource with the
      * given name, which may or may not exist.
+     * <p>
+     *     Constraints:
+     *     <ul>
+     *         <li>The path is expected to be in URI path-like format.</li>
+     *         <li>The path provided will not be URI decoded by the Resource implementation.</li>
+     *         <li>URI canonicalization will be attempted before being used against the Resource.
+     *             This can result in a normal Resource, perhaps with it being an alias to a different Resource,
+     *             perhaps even rejected with an Exception if there is an attempt to navigate above this Resource.</li>
+     *         <li>The path provided will be given to the Resource implementation for interpretation on how it
+     *             will behave (such as using "a\\b.txt" on a Path based in linux vs windows).</li>
```

Comment:
Hmmm probably true... but really not very desirable.
Ideally `\` would never be treated as a separator as the separator is '/' in this space.